### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by Sddilora** [Ubn-Jr/hirob-be-core#14] [General]
Introducing yourself

**Comment by furkanadiiguzel** [comment]
Hello everyone,

I'm Furkan Adıgüzel, and I'm currently a junior student majoring in Information Systems and Technologies at Bilkent University. I'm excited to be a part of this project as a Backend Developer. Our role includes ensuring data integrity and security, providing seamless integration, and maintaining high availability and performance. We are also dedicated to implementing scalable infrastructure, facilitating CI/CD processes, staying updated, and fostering collaboration.

Feel free to reach out to me if you have any questions or ideas related to project. I'm always eager to learn and contribute to the success of the project. Let's work together to make this project a great success!

**Comment by johsquaree** [comment]
Hello everyone,
I'm  Furkan Kayapinar, and I'm currently actively involved in the backend developer and computer engineering aspects of my software career. Simultaneously, I'm contributing to open-source code and continuing my computer engineering education at Beykent University.
I'm also part of our team's backend section to bring our ideas to life for this project.

If you have any questions or ideas related to the project, please feel free to reach out to me. I'm always eager to learn and contribute to the project's success. Let's work together to make this project a great success!
